src/components/BaseMenu/BaseMenu.tsx

```diff
@@ -0,0 +1,52 @@
+import { useRef, useState } from 'react';
+import BaseButton from '../BaseButton';
+import iconMenu from '../../assets/icon-vertical-ellipsis.svg';
+import './BaseMenu.css';
+import useClickOutside from '../../hooks/useClickOutside';
+
+export enum MenuOptionColor {
+  default = 'default',
```

Comment:
Mantener nomenclatura capitalizada para las propiedades de los enumeradores.
```suggestion
  Default = 'default',
```